bundles/org.openhab.ui/web/src/pages/settings/things/link/link-add.vue

```diff
@@ -19,31 +19,36 @@
         </f7-list>
       </f7-col>
 
-      <!-- Option to create new item (if not supplied by prop) -->
-      <f7-col v-if="!item">
-        <f7-block-title>Item</f7-block-title>
-        <f7-list media-list>
-          <f7-list-item radio :checked="createItem === false" value="false" @change="createItem = false" title="Use an existing Item" name="item-creation-choice" />
-          <f7-list-item radio :checked="createItem === true" value="true" @change="createItem = true" title="Create a new Item" name="item-creation-choice" />
-        </f7-list>
-      </f7-col>
+      <div v-if="!item && !items" class="text-align-center">
```

Comment:
There's another preloader block, lines 77-80, conditionally displayed when `!ready` so you should either remove this one or the other (presumably the other) and alter its v-if or control `ready` to cover all readiness conditions.

Right now the two can be seen simulateously for a few milliseconds which is not a good look. If the item-form were displayed you would even have a third one.

Reply:
the second preloader is now only shown if the first isn't